pageserver/src/context.rs

```diff
@@ -109,6 +111,67 @@ pub enum DownloadBehavior {
     Error,
 }
 
+/// Whether this request should update access times used in LRU eviction
+#[derive(Clone, Copy, PartialEq, Eq, Debug)]
+pub enum ATimeBehavior {
```

Comment:
If everything is always `pub`, we will never get any unused code warnings.

```suggestion
pub(crate) enum ATimeBehavior {
```

While I am not sure if we should get away from all uses `bool` for clearly `if update_access_stats { ... }`, this at least should not be called `atime` because we've often times discussed use of fs atime, and we are currently using `mtime` for trashing warnings.

```suggestion
pub(crate) enum UpdateAccessStats {
```

But this could be a boolean.